GVFS/GVFS.Build/GVFS.PreBuild.csproj

```diff
@@ -162,7 +162,6 @@
   <ItemDefinitionGroup>
     <ApplicationNeedsManifest>
       <Visible>false</Visible>
-      <OutputFile>$(BuildOutputDir)\%(Identity).exe.manifest</OutputFile>
     </ApplicationNeedsManifest>

```

Comment:
I am not sure we need this `ItemDefinitionGroup` at all with this change (or maybe it is used elsewhere...)...

Reply:
Might want to test if removing it causes the .exe.manifest to show as a child of project in the VS solution hierarchy. If it doesn't then we can remove it.

Reply:
ah - that it does (well, it causes a `GVFS.service` item to appear). Will keep it!